src/ClientImpl.cpp

```diff
@@ -378,7 +378,8 @@ void ClientImpl::initiateAuthentication(PendingConnection* pc, struct buffereven
         throw voltdb::LibEventException();
     }
     AuthenticationRequest authRequest( m_username, "database", m_passwordHash, m_hashScheme );
-    ScopedByteBuffer bb(authRequest.getSerializedSize());
+    int size = authRequest.getSerializedSize();
```

Comment:
If we leave it in we get a warning from the compiler that size is never used.  I thought it was better to avoid the warning.